JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -863,7 +863,7 @@ - (void)preferredContentSizeChanged:(NSNotification *)notification
 - (void)jsq_updateCollectionViewInsets
 {
     const CGFloat top = self.additionalContentInset.top;
-    const CGFloat bottom = CGRectGetMaxY(self.collectionView.frame) - CGRectGetMinY(self.inputToolbar.frame) + self.additionalContentInset.bottom;
+    const CGFloat bottom = self.inputToolbar.frame.size.height + self.additionalContentInset.bottom;
```

Comment:
Btw I'm also not very happy about this change but that's the best I was able to get.